src/server.c

```diff
@@ -1140,6 +1187,16 @@ void nvnc_close(struct nvnc* self)
 	LIST_FOREACH_SAFE (client, &self->clients, link, tmp)
 		client_unref(client);
 
+	struct sockaddr_storage addr;
```

Comment:
I think that this code block can be extracted to a function to keep `nvnc_close` short and easy to read.